test/e2e/common/runtimeclass.go

```diff
@@ -97,16 +97,9 @@ func createRuntimeClass(f *framework.Framework, name, handler string) string {
 }
 
 func expectPodRejection(f *framework.Framework, pod *v1.Pod) {
-	// The Node E2E doesn't run the RuntimeClass admission controller, so we expect the rejection to
-	// happen by the Kubelet.
```

Comment:
This condition code has been added since the commit 2c01dc0ebfcba5e9d6aabd19aae07fe0d76d1bc8
It is nice to get feedback from @tallclair 

/cc @oomichi

Reply:
/cc  @tallclair